tag-release

```diff
@@ -52,7 +52,7 @@ for REPO in ${REPOS}; do
   git tag -d "$TAG" || echo "No local tags deleted"
   git push --delete origin "$TAG" || echo "No remote tags deleted"
   # Check if we have to update the submodules while tagging
-  if [ "${REPO}" = "scripts" ] && [ -d "sdk_container/src/third_party/" ] ; then
+  if [ "${REPO}" = "scripts" ] && [ -d "sdk_container/src/third_party/" ] && [ "$CHANNEL" != "lts" ] ; then
```

Comment:
```suggestion
  if [ "${REPO}" = "scripts" ] && [ "$CHANNEL" != "lts" ] ; then
```
The directory check doesn't make sense anyway because it assumes that a meaningful checkout took place which is not the case that this only would happen when the check succeeds^^